dockerfiles/cli/cli.sh

```diff
@@ -832,18 +834,15 @@ cmd_config() {
   fi
 
   # Run the docker configurator
-  generate_configuration_with_puppet 
+  if [ "${CODENVY_DEVELOPMENT_MODE}" = "on" ]; then
+    # generate configs and print puppet output logs to console if dev mode is on
+    generate_configuration_with_puppet
+  else
+    generate_configuration_with_puppet >> "${LOGS}"
+  fi
```

Comment:
Why did you add this back in?  I removed this "if" logic earlier because it was suppressing certain output that was important.